Dockerfile

```diff
@@ -15,8 +15,9 @@ COPY package.json /open-oracle/package.json
 RUN yarn install
 
 ENV PROVIDER PROVIDER
-ADD contracts contracts
-ADD saddle.config.js saddle.config.js
+COPY contracts contracts
+COPY tests tests
+COPY saddle.config.js saddle.config.js
```

Comment:
Still 3 layers - can't this be `COPY contracts tests saddle.config.js .`?

Reply:
not really, such copy statement will copy content of dirs, but not directories itself. We can do COPY . ., but then .dockerignore should be extended significantly.

Reply:
but so why not just use the dockerignore?

Reply:
FWIW, we probably should make sure that dockerignore is up to date, since it sends the "build context" and adds a lot of time to building docker images if it includes unnecessary resources.